src/main/java/uk/gov/ons/ctp/response/casesvc/endpoint/CaseEndpoint.java

```diff
@@ -132,16 +132,28 @@ public CaseEndpoint(
    * @param partyId to find by
    * @param caseevents flag used to return or not CaseEvents
    * @param iac flag used to return or not the iac
+   * @param max_cases_per_survey the maximum number of collection exercises to return per survey
    * @return the cases found
    */
-  @Deprecated // See findCases(sampleUnitId, partyId)
```

Comment:
Why are we un-deprecating this method? Surely we should be modifying `findCases(sampleUnitId, partyId)` instead?

Reply:
In the commit that added it Ben mentions it was only use was in tests , and no mention of why he did not use in in actual code .  Given this was over a year ago and the existing endpoint was in use It seemed a little outside of the scope this task to implement his deprecation ?

Reply:
Sure, that's fine with me